core/src/main/java/com/uber/m3/tally/ValueBuckets.java

```diff
@@ -140,7 +140,7 @@ public static ValueBuckets exponential(double start, double factor, int numBucke
 
         Double[] buckets = new Double[numBuckets];
```

Comment:
Can we create a double[] instead and box all of them later? If the intent of this change was the concern around boxing of double.

`    Double[] boxedBuckets = Arrays.stream(buckets).boxed().toArray(Double[]::new);
`